docs/sources/administration/data-source-management/index.md

```diff
@@ -81,27 +82,29 @@ You can assign data source permissions to users, teams, and roles which will all
 
 <div class="clearfix"></div>
 
-## Query caching
+## Query and resource caching
 
-When query caching is enabled, Grafana temporarily stores the results of data source queries. When you or another user submit the exact same query again, the results will come back from the cache instead of from the data source (like Splunk or ServiceNow) itself.
+When query and resource caching is enabled, Grafana temporarily stores the results of data source queries and resource requests. When you or another user submit the exact same query or resource request again, the results will come back from the cache instead of from the data source itself.
 
-Query caching works for all backend data sources. You can enable the cache globally and configure the cache duration (also called Time to Live, or TTL).
+A "query" refers to a data source query for data frames to be transformed and/or visualized in Grafana. A "resource" refers to any HTTP requests a plugin might make, for example: the Amazon Timestream plugin requesting a list of available databases from AWS. For more information on data source queries and resources, please see the developers page on [backend plugins]({{< relref "../../developers/plugins/backend/" >}}).
+
+The caching feature works for **all** backend data sources. You can enable the cache globally in Grafana's [configuration]({{< relref "../../setup-grafana/configure-grafana/enterprise-configuration/#caching" >}}), and configure a cache duration (also called Time to Live, or TTL) for each data source individually.
 
 {{% admonition type="note" %}}
 Available in [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
 {{% /admonition %}}
 
-The following cache backends are available: in-memory, Redis, and Memcached.
+The following cache backend integrations are available: in-memory, Redis, and Memcached.
```

Comment:
What's the reasoning behind adding `integrations` here?

Reply:
no strong reason - I just thought it sounded a bit more natural. I could also say "backend implementations" or just "backend options" if one of those seems better.

Reply:
Oh I like `backend options`, if that works for you!

Reply:
Done [64f1100](https://github.com/grafana/grafana/pull/70556/commits/64f1100386bdd1390bf01a09ab9e6175f9fb9beb)